content/en/tracing/guide/ddsketch_trace_metrics.md

```diff
@@ -0,0 +1,48 @@
+---
+title: DDSketch based Metrics in APM
+kind: guide
+aliases:
+- /tracing/faq/resource-trace-doesn-t-show-up-under-correct-service/
+further_reading:
+- link: "https://www.datadoghq.com/blog/engineering/computing-accurate-percentiles-with-ddsketch/"
+  tag: "Blog"
+  text: "Computing Accurate Percentiles with DDSketch"
+- link: "https://docs.datadoghq.com/metrics/distributions/"
+  tag: "Documentation"
+  text: "Learn more about Distributions"
+- link: "https://docs.datadoghq.com/tracing/guide/metrics_namespace/"
+  tag: "Documentation"
+  text: "Learn more about Trace Metrics"
+---
+
```

Comment:
Since a lot of this guide talks about something in beta, let's put the standard alert box up here, something like...

```
<div class="alert alert-warning">
DDSketch trace metrics are in beta. Request access by...
 </div>
```